app/components/renderWhitespaceCharacters/renderWhitespaceCharacters.js

```diff
@@ -0,0 +1,52 @@
+(function() {
+  'use strict';
+
+  /**
+   * @name display-character
+   * @description display whitespace character with symbol(HTML),
+   *              *NOTE*, need to wrap around <pre> tag
+   * @ngInject
+   */
+  function renderWhitespaceCharacters() {
+    var WHITESPACES = {
+      'whitespace' : {
```

Comment:
I think we should just call this 'space' so that it does not overload the term "whitespace"